### PR TITLE
reword first init prompt

### PR DESCRIPTION
**Pull request by FredKSchott**
Running through some workflows, the current wording on the template select prompt felt awkard to me. Some of the descriptions didn't seem to do their templates justice either. 

This PR rewrites them to better explain what each template does and why you would choose it. Happy to bikeshed on any of them, as this was just my best attempt at each.

/cc @justinfagnani @azakus

### CODE REVIEW

File: src/commands/init.ts [hunk 1]
```diff
@@ -47,11 +47,10 @@ export class InitCommand implements Command {
...
 
       let templateDescriptions = {
-        'application': "Application-style project",
-        'element': "Reusable element-style project",
-        'shop': "The Shop PRPL demo application",
-        'app-drawer-template':
-            "Application template that demonstrates the PRPL pattern",
+        'element': 'A blank element template',
+        'application': 'A blank application template',
+        'app-drawer-template': 'A starter application template with a drawer-based layout',
```
Comment (justinfagnani) [inline src/commands/init.ts]: I think it's somewhat important to keep PRPL in the app-drawer description

Comment (FredKSchott) [inline src/commands/init.ts]: done

**Comment by FredKSchott** [comment]
@justinfagnani reworded app-drawer-template to mention PRPL, PTAL

**Comment by azakus** [comment]
LGTM

File: src/commands/init.ts [hunk 2]
```diff
@@ -47,11 +47,10 @@ export class InitCommand implements Command {
...
       let templateDescriptions = {
-        'application': "Application-style project",
-        'element': "Reusable element-style project",
-        'shop': "The Shop PRPL demo application",
-        'app-drawer-template':
-            "Application template that demonstrates the PRPL pattern",
+        'element': 'A blank element template',
+        'application': 'A blank application template',
+        'app-drawer-template': 'A starter application template, with navigation and "PRPL pattern" loading',
+        'shop': 'The "Shop" Progressive Web App demo',
```
Comment (justinfagnani) [inline src/commands/init.ts]: Shop should mention PRPL too

**Pull request merged**